standard/documentation-comments.md

```diff
@@ -35,13 +35,15 @@ In a *Delimited_Doc_Comment*, if the first non-*Whitespace* character on the sec
 **Example:**
 
 ```csharp
-/// <summary>Class <c>Point</c> models a point in a two-dimensional
-/// plane.</summary>
-///
+/// <summary>
+/// Class <c>Point</c> models a point in a two-dimensional plane.
+/// </summary>
 public class Point
 {
-   /// <summary>method <c>draw</c> renders the point.</summary>
-   void draw() {...}
+    /// <summary>
+    /// method <c>draw</c> renders the point.
```

Comment:
Capitalize the "m" of method and rename the method to Draw to follow normal conventions? (We could do that sort of thing as a separate pass, certainly.)

Reply:
I agree on "m".  Because `draw()` is private, I thought it should stay lowercase. Thoughts?

Reply:
Hmm... I've always used a convention of making all methods start with a capital letter, regardless of whether they're public or private. It's irrelevant from a public API perspective, of course. You've probably worked in more codebases than me - what's your experience of the common convention?

Reply:
I just checked the [runtime coding guidelines](https://github.com/dotnet/runtime/blob/main/docs/coding-guidelines/coding-style.md). See item 12. They use `PascalCase` for all methods. Let's follow that convention.